src/main/java/step4/Main.java

```diff
@@ -0,0 +1,54 @@
+package step4;
+
+import step4.VO.Lottos;
+import step4.VO.ManualLottoCnt;
+import step4.VO.ManualLottoNumbers;
+import step4.VO.Money;
+import step4.View.InputView;
+import step4.View.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+    public void main(String[] args) {
+        // 금액 입력
+        Money money = new Money(InputView.getMoney());
+        // 로또 생성기 선언
+        LottoGenerator lottoGenerator = new LottoGenerator();
+
+        Lottos lottos = new Lottos();
+
+        // 수동 발권 로직 시작
+        // 수동 발권 갯수 입력
+        ManualLottoCnt manualCnt = new ManualLottoCnt(InputView.getManualLotto(), money);
+
+        List<ManualLottoNumbers> manualLottoNumbersList = new ArrayList<>();
+        for(int i = 0; i < manualCnt.getManualLottoCnt(); i++) {
+            // 수동 발권 번호 입력
+            manualLottoNumbersList.add(new ManualLottoNumbers(InputView.getManualLottoNumbers()));
+        }
+
+        lottos.buyManualLottos(manualLottoNumbersList);
+        buyAutoLottos(lottos, money, lottoGenerator);
+
+        ResultNumber resultNumber = new ResultNumber();
+
+        resultNumber.settingLottoResultNumber(InputView.getResultNumber());
+        resultNumber.settingLottoBonusNumber(InputView.getBonusNumber());
+
+        Statistic statistic = new Statistic();
+        lottos.checkResult(statistic, resultNumber);
```

Comment:
문맥을 고려해봤을때 다음과 같이 변경된다면 객체에게 메시지를 보내는 문맥이 자연스러워질 것 같네요!
```suggestion
        Statistic statistic = lottos.analyzeWin(resultNumber);
```

Reply:
구조상 로또 1장 결과를 확인해도 statistic에 반영될 수 있도록 일부러 lotto안에 넣고 return 없이 void로 설계하였습니다